pkg/distributor/errors.go

```diff
@@ -287,7 +316,16 @@ func wrapPartitionPushError(err error, partitionID int32) error {
 		return nil
 	}
 
-	return errors.Wrap(err, fmt.Sprintf("%s %d", failedPushingToPartitionMessage, partitionID))
+	// Add the partition ID to the error message.
+	err = errors.Wrap(err, fmt.Sprintf("%s %d", failedPushingToPartitionMessage, partitionID))
```

Comment:
nit: This is returned to user as well. Is partition ID relevant for this error? (we already return partition IDs to user, so it's not big deal, just wondering)

Reply:
First of all this hasn't changed in this PR. It's useful for us, not the final user. But it's very handy for us including it here, because then it gets logged.